testprojects/src/python/coordinated_runs/phaser.py

```diff
@@ -0,0 +1,14 @@
+import os
+import sys
+import time
+
+
+arrive_file, await_file = sys.argv[1:3]
+
+# Signal we've arrived.
+with open(arrive_file, 'w') as fp:
+  fp.close()
```

Comment:
Just a question -- does "phaser" refer to some construction/method I'm not familiar with? I haven't heard the term before.

Reply:
From this thing: https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/Phaser.html